### PR TITLE
[sw/silicon_creator] Rename keyset to key_struct

### PR DESCRIPTION
**Pull request by alphan**

### CODE REVIEW

**Comment by alphan** [comment]
@cfrantz this PR replaces "set" with "duplet" so that we are free to use the term "keyset" for the actual set of keys.

File: rules/opentitan.bzl [hunk 1]
```diff
@@ -86,7 +86,7 @@ def create_prod_key(name, label):
         CONST.LCV.RMA,
     ])
 
-def create_keyset(rsa_key, spx_key):
+def create_key_duplet(rsa_key, spx_key):
```
Comment (cfrantz) [inline rules/opentitan.bzl:89]: This is purely personal preference, but I don't like the term "duplet" and we'll have to change it if ever becomes more than two things.

I think the word "tuple" or "struct" would fit a bit better.
- Tuple may have some implied meaning given starlark's pythonic legacy (ie: an immutable sequence).
- The entity that is created is actually a struct.

**Review by cfrantz — APPROVED** [review]

File: rules/opentitan.bzl (hunk 1 above)
Comment (alphan) [reply rules/opentitan.bzl:89]: same here :) I didn't want to use `pair` or `tuple` for the same reasons. I'm ok with `struct`. I'll update and merge, thanks!

**Review by moidx — APPROVED** [review]

**Comment by alphan** [comment]
@timothytrippel fyi, changes the names of some definitions there should be no changes in outputs.

**Comment by timothytrippel** [comment]
LGTM, should not impact DV.

**Pull request merged**